dessin-pdf/src/lib.rs

```diff
@@ -1,21 +1,28 @@
+use dessin::font::FontRef;
 use dessin::{
     export::{Export, Exporter},
-    font::FontGroup,
+    font::{get, Font, FontGroup, FontHolder},
```

Comment:
Pour evité les problème dans le future, il ne faudrai pas que font::get soit import. get est un nom trop générique.

```suggestion
    font::{self, Font, FontGroup, FontHolder},
```

Donc dans l'utilisation de la fonction, il faudrai plutôt écrire `font::get`